optimum/neuron/utils/testing_utils.py

```diff
@@ -33,6 +36,12 @@ def requires_neuron_or_neuronx(test_case):
     )(test_case)
 
 
+def requires_pytorch_1_13(test_case):
+    return unittest.skipUnless(
+        version.parse(get_torch_version()) < version.parse("2.0.0"), "test requires PyTorch < 2.0.0"
+    )(test_case)
+
+
```

Comment:
Why?
I think once we move to PT 2.* we should drop the support for PT 1.* altogether.

Reply:
It's was a temp workaroud for passing the test since by the time of neuron SDK 2.17, the weights/neff replacement was only supported for pt1.13 not pt 2.1: https://github.com/aws-neuron/aws-neuron-sdk/issues/846

I will test today to check if this has been patched by the 2.18 SDK. If so we can remove this.

Reply:
It's patched, will remove useless!